docs/tutorial-dev-kinto-admin.rst

```diff
@@ -149,10 +149,10 @@ Development `happens on Github <https://github.com/Kinto/kinto-admin>`_.
 The process for a patch to reach Remote Settings is the following:
 
 * Get the patch merged on Kinto/kinto-admin
-* Release a new version of `kinto-admin on NPM <https://www.npmjs.com/package/kinto-admin>`_
-* Upgrade the kinto-admin plugin in Kinto (`example PR <https://github.com/Kinto/kinto/pull/2255>`_)
-* Release a new version of Kinto
-* Upgrade Kinto in `Remote Settings <https://github.com/mozilla/remote-settings/>`_
+* Create a new release version of kinto-admin on Github (`kinto-admin releases <https://github.com/Kinto/kinto-admin/releases>`_)
+* Upgrade the kinto-admin package in Kinto (`example kinto PR <https://github.com/Kinto/kinto/pull/3303>`_)
```

Comment:
Since it's not pushed on NPM anymore, I would not really call it a *package*, but just a *Git tag*?

Reply:
Changed it to "release" as I suppose that's what we really want people to pull